shipstation_integration/setup.py

```diff
@@ -244,46 +244,73 @@ def setup_custom_fields(args=None):
 				label="Shipstation Shipment ID",
 				insert_after="shipstation_order_id",
 				translatable=False,
-			),
-			dict(
-				fieldtype="Section Break",
-				fieldname="sb_shipment",
-				collapsible=True,
-				label="Shipment Details",
-				insert_after="has_pii",
-			),
-			dict(
-				fieldtype="Data",
-				fieldname="carrier",
-				read_only=True,
-				label="Carrier",
-				insert_after="sb_shipment",
-				translatable=False,
-			),
-			dict(
-				fieldtype="Data",
-				fieldname="tracking_number",
-				read_only=True,
-				label="Tracking Number",
-				insert_after="carrier",
-				translatable=False,
-			),
-			dict(
-				fieldtype="Column Break",
-				fieldname="cb_shipment",
-				insert_after="tracking_number",
-			),
-			dict(
-				fieldtype="Data",
-				fieldname="carrier_service",
-				read_only=True,
-				label="Carrier Service",
-				insert_after="cb_shipment",
-				translatable=False,
-			),
+			)
 		]
 	)
 
+	shipment_fields = [
+		dict(
+			fieldtype="Data",
+			fieldname="shipstation_store_name",
+			read_only=True,
+			label="Shipstation Store",
+			insert_after="shipment_amount",
+			translatable=False,
+		),
+		dict(
+			fieldtype="Data",
+			fieldname="shipstation_order_id",
+			read_only=True,
+			label="Shipstation Order ID",
+			insert_after="shipstation_store_name",
+			in_standard_filter=True,
+			translatable=False,
+		),
+		dict(
+			fieldtype="Data",
+			fieldname="marketplace",
+			read_only=True,
+			label="Marketplace",
+			insert_after="awb_number",
+			translatable=False,
+		),
+		dict(
+			fieldtype="Data",
+			fieldname="marketplace_order_id",
+			read_only=True,
+			label="Marketplace Order ID",
+			insert_after="marketplace",
+			translatable=False,
+		)
+	]
+
+	shipment_parcel_fields = [
```

Comment:
If you want to do this, then I'd suggest using Property Setters to change the label value instead of adding new fields. I think the Parsimony app is handling that in its `setup.py`.

Reply:
I pushed this by mistake, I was just checking this way but this fails as these fields already exists. Will add property setters.